server/protocol.go

```diff
@@ -86,15 +84,23 @@ func (s *protocolServer) Propose(ctx context.Context, req *curppb.ProposeRequest
 		}
 		return nil, res.Error.Err()
 	}
+
+	s.proposeOpCnt++
+	switch s.proposeOpCnt {
+	case 1:
+		err := RpcError{Code: 3, Message: "shutting down"}
+		return nil, err.Err()
+	case 2, 3, 4:
+		err := RpcError{Code: 3, Message: "key conflict"}
```

Comment:
Please check the status Code in xline, `3` means `INVALID_ARGUMENT`

Reply:
I changed it to `13` means `Internal`.